charts/apisix-ingress-controller/values.yaml

```
@@ -209,3 +209,19 @@ securityContext: {}
   # readOnlyRootFilesystem: true
   # runAsNonRoot: true
   # runAsUser: 1000
+
```

README file is auto-generated from field comments. Also a newline at eof is good.
```suggestion
gateway:
  # -- Apache APISIX service type for user access itself
  type: NodePort
  externalTrafficPolicy: Cluster
  # annotations:
  #   service.beta.kubernetes.io/aws-load-balancer-type: nlb
  # loadBalancerIP: a.b.c.d
  # loadBalancerSourceRanges:
  #   - "143.231.0.0/16"
  # -- load balancer ips
  externalIPs: []
  nginx:
    # -- Nginx workerRlimitNoFile
    workerRlimitNofile: "20480"
    # -- Nginx worker connections
    workerConnections: "10620"
    # -- Nginx worker processes
    workerProcesses: auto
    # -- Nginx error logs path
    errorLog: stderr
    # -- Nginx error logs level
    errorLogLevel: warn

```